src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -117,6 +113,15 @@ protected override void OnScriptFileChanged(FileSystemEventArgs e)
             }
         }
 
+        private void Shutdown()
```

Comment:
Decided to rename the method to be a more descriptive of what it does for now. I do have plans to change how we're dealing with this shutdown/restart actions in the future leveraging the eventing stuff we've just added.

Let me know what you think...